lumicks/pylake/kymotracker/kymotrack.py

```diff
@@ -7,6 +7,70 @@
 from ..population.dwelltime import DwelltimeModel
 
 
+def _extract_dwelltime_data_from_groups(groups, exclude_ambiguous_dwells):
```

Comment:
I don't really like this at the module level. I think it's distracting, especially for a function that's only used in one place. I get that it makes sense here in that it doesn't use `self`, but I think I'd prefer it as a method on `KymoTrackGroup` and then can call `_tracks_by_kymo()` internally rather than passing in `groups`.

Reply:
Yeah, fair enough. Initially I was doubting whether this function should actually live here, or in the dwelltime module, but now that it's here, I agree it'd be nicer to stick it in the class where it belongs.

I'd prefer to keep it a static method though. If all its doing is looking up the tracks immediately with `_tracks_by_kymo()`, I'd rather have that a bit more explicit. It also makes the test a bit more readable imho.